app/resources/server/model.go

```diff
@@ -27,12 +27,13 @@ type All struct {
 type Essential struct {
 	IP         string `json:"ip"`
 	Hostname   string `json:"hn"`
-	Players    int    `json:"pc"`
-	MaxPlayers int    `json:"pm"`
+	Players    int64  `json:"pc"`
```

Comment:
bro is expecting over 2 billion players 💀

Reply:
due to incompatibility and different datatype naming in cockroachdb and postgresql, I had to change it from int to in64 to support the BigInt value in Prisma, because just "Int" wasn't working